### ISSUE TITLE
third-slot voting points value same as second-slot

### ISSUE THREAD

**Issue by zedwick** [stefvanschie/buildinggame#76]
When voting, selecting the third slot will vote 2 points, and selecting the second slot will also vote 2 points. All other voting slots are as expected.

My config is set up as follows:

```
voting:
  default-vote-points: 5
  second-slot:
    id: coal_block
    points: 2
  third-slot:
    id: iron_block
    points: 3
  fourth-slot:
    id: lapis_block
    points: 4
  fifth-slot:
    id: redstone_block
    points: 5
  sixth-slot:
    id: gold_block
    points: 6
  seventh-slot:
    id: diamond_block
    points: 7
  eighth-slot:
    id: emerald_block
    points: 8
```

buildinggame version: 5.0.1
Paper 1.12
Players: 3

**Comment by stefvanschie** [comment]
Verified, will be fixed in the next version.

**Comment by stefvanschie** [comment]
Fixed in version 5.0.2.
